data/language/en-GB.txt

```diff
@@ -4321,6 +4321,8 @@ STR_6009    :{SMALLFONT}{BLACK}Select cut height
 STR_6010    :{COMMA2DP32}m
 STR_6011    :{COMMA1DP16}ft
 STR_6012    :{COMMA1DP16}
+STR_6013    :Rides don't decrease in value over time
```

Comment:
The problem with that is that this cheat doesn't prevent a ride from getting older, it prevents it from depreciating.

I'd change it to `Prevent rides from losing value`.